Documentation/guides/messages/xa0128.md

```diff
@@ -0,0 +1,26 @@
+---
+title: Xamarin.Android error XA0128
+description: XA0128 error code
+ms.date: 08/07/2020
+---
+# Xamarin.Android error XA0128
+
+## Issue
+
+The Fast Deployment system relys on reading the output from the various system applications
+on the device. This is done by reading `System.out`, which by default is where all output
+will be redirected. If `log.redirect-stdio` this will cause all of the Fast Deployment tooling
+to fail.
+
+## Solution
+
+Disable the redirection of stdio in order to use Fast Deployment. This can be done using
+the following commands.
+
+```
+adb shell stop
+adb shell setprop log.redirect-stdio false
+adb shell start
```

Comment:
Is there a way to use `@(AndroidEnvironment)` to do this? It might be easier for users if it works.

Reply:
not really. Because on devices you need to run this as `root`. So it needs to be run by the user.
To be honest, I've never seen anyone report this issue so I'm not sure if anyone actually changes the android `log.redirect-stdio`  property.